Update osgusertraining.md (#165)
osg-htc/user-documentation: documentation/support_and_training/training/osgusertraining.md

</details>

<details>
<summary>[Webinar] Move Your Data with Pelican (and the OSDF)</summary>
<br>
<b>Learning Objectives</b>
<br>

> Pelican is a platform created to enable easier data sharing - within or beyond your institution! This training will cover how Pelican is used to move data within the OSPool and also how you can use Pelican tools to host, upload and download your data. We encourage researchers with large amounts of data, as well as campus representatives, to attend to learn about how Pelican can help with your data movement needs.

Reviewer: ```suggestion
Pelican is a platform created to enable easier data sharing - within or beyond your institution! This training will cover how Pelican is used to move data within the OSPool and also how you can use Pelican tools to host, upload and download your data. This training is relevant for researchers with large amounts of data, as well as campus representatives, to learn about how Pelican can help with your data movement needs.
```
